sdk/monitor/monitor-query/src/logsQueryClient.ts

```diff
@@ -103,10 +103,12 @@ export class LogsQueryClient {
       }
     );
 
+    const parsedBody = JSON.parse(rawResponse.bodyAsText!);
+    flatResponse.tables = parsedBody.tables;
```

Comment:
@KarishmaGhiya This fix is not needed for the migration to core v2 but is crucial to fix a bug in the service response. I am happy that you mastered the art of hacking the raw response to fix customer-facing bugs!